src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.scss

```diff
@@ -75,8 +89,10 @@
   margin-top: 30px;
 }
 
-.actions-cell button:not(.dropdown-item) {
-  margin-left: var(--btn-margin);
+@media (min-width: 768px) { //applies only to non-mobile screens
+  .actions-cell button:not(.dropdown-item) {
+    margin-left: var(--btn-margin);
+  }
```

Comment:
@athakaras @weiquu Not really proficient in advanced CSS, may I ask where is `--btn-margin` coming from? Or rather what is the value of this variable?

Reply:
@domlimm I believe it's from `styles.scss` where it was set with:
```
:root {
  --btn-margin: 5px;
}
```

Can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/var)!